internal/service/common/clients/resources/oapi-codegen.yaml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/oapi-codegen/oapi-codegen/HEAD/configuration-schema.json
+package: resources
+output: client.gen.go
```

Comment:
shouldnt this whole thing be under `alarms` dir since we are the client of resource server?

Reply:
sort of depends on how we look at this... if we had 2 clients wanting to use it then we'd likely want to put in under `resources`... but for now only `alarms` is a user so it seems better there because of that.    In the end it doesn't matter much until there is a 2nd client.

Reply:
moved it within alarms directory